app/src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -995,6 +998,102 @@ class StateFragmentLocalTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "port")
+  fun testStateFragment_mobilePortrait_finishExploration_endOfSessionConfettiIsDisplayed() {
```

Comment:
Suggest also adding a test for configuration changes on the end exploration (to verify that it replaces as expected).

Reply:
Done